src/pages/Ingredient/IngredientList.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import './IngredientList.scss';
+
+const SLIDE_LEFT_DIRECTION = 'left';
+const SLIDE_RIGHT_DIRECTION = 'right';
+
+function IngredientList({ product }) {
+  const [xPos, setXpos] = useState(0);
+
+  const clickLeftorRight = direction => {
+    direction === SLIDE_LEFT_DIRECTION
+      ? setXpos(x => x + 100)
+      : setXpos(x => x - 100);
+    if (xPos === 300 || xPos === -600) return setXpos(0);
```

Comment:
- 얼리 리턴 패턴은 if 문이 가장 먼저 와야 의미가 있습니다 성재님-!
```suggestion
    if (xPos === 300 || xPos === -600) return setXpos(0);
    
    direction === SLIDE_LEFT_DIRECTION
      ? setXpos(x => x + 100)
      : setXpos(x => x - 100);
```